src/components/WorkshopPage/Workshop.js

```diff
@@ -3,42 +3,88 @@ import ReactPlayer from 'react-player/youtube';
 import Button from '@material-ui/core/Button';
 import Grid from '@material-ui/core/Grid';
 import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import YouTubeIcon from '@material-ui/icons/YouTube';
+import GitHubIcon from '@material-ui/icons/GitHub';
+import SlideshowIcon from '@material-ui/icons/Slideshow';
 
-function Workshop(props) {
+const useStyles = makeStyles(theme => ({
+	button: {
+		textTransform: 'none',
+		padding: '4px 1.5em',
+		maxWidth: 'fit-content'
+	},
+	video: {
+		borderRadius: '10px',
+		overflow: 'hidden'
+	},
+	title: {
+		marginTop: '1em'
```

Comment:
Same here for consistency, we used `em` to specify margin, but in the `icon` class, we used `theme.spacing`. I did some reading - apparently `em` will change depending on the `fontSize` property in the same class. Let's just use material-ui's `theme.spacing` for consistency (I'm actually no 100% sure which one we're supposed to use, same with padding).

Reply:
So Jody and I actually were looking at the inconsistency between the use of em, px, and theme.spacing and we both agreed that it was a little inconsistent. I think we both decided that the work of redoing them with theme.spacing might be a little much, so I didn't change the padding to theme.spacing. I'll definitely work on changing it, but I just wanted to let you know why we kept the padding like this so far.